tools/crawler/urls.txt

```diff
@@ -0,0 +1,517 @@
+http://google.com
```

Comment:
Maybe drop the protocol from the URLs?

Reply:
OK, if it doesn't start with `https?` it will add `http`